packages/store-indexer/src/bin/sqlite-indexer.ts

```diff
@@ -136,7 +143,7 @@ server.use(
   }),
 );
 server.use(helloWorld());
-server.use(apiRoutes(database));
+server.use(apiRoutes({ database, enableUnsafeQueryApi: env.ENABLE_UNSAFE_QUERY_API }));
```

Comment:
should we print a fat `WARNING: unsafe query API is enabled, DO NOT USE IN PRODUCTION as it will lead to public database access` or something similar to the console if this is enabled? i know some people are using the sqlite indexer in prod, don't want them to enable this because they're like "oh cool sql api is available for sqlite indexer now"

Reply:
yeah very valid point, added the warning message (attached screenshot in PR description)

Reply:
:chefkiss: